### PR TITLE
Remove unnecessary commands from connect_to_rhel_console.exp file

### PR DESCRIPTION
**Pull request by ksimon1**
remove unnecessary commands from connect_to_rhel_console.exp file

Signed-off-by: Karel Šimon <ksimon@redhat.com>

### CODE REVIEW

**Comment by fromanirh** [comment]
Looks good, but please explain why the commands are unnecessary, and, if possible, since when they are unnecessary.

**Comment by MarSik** [comment]
Yep, what @fromanirh said. The patch is good, just improve the commit message to explain what you did there (removed duplicated timeout loop, fixed too short default timeout of expect)...

**Comment by ksimon1** [comment]
@MarSik, @fromanirh done

**Comment by fromanirh** [comment]
Thanks! Almost there. Why 300 seconds? Is just a "big enough" value? In general a good commit message explains "the why" and provides context and backstory for the change. The "how" is usually self-evident from the code.
Everything else looks ok.

**Comment by ksimon1** [comment]
@fromanirh it is default timeout defined here: https://github.com/kubevirt/common-templates/blob/master/automation/test-rhel.sh#L55

**Comment by fromanirh** [comment]
> @fromanirh it is default timeout defined here: https://github.com/kubevirt/common-templates/blob/master/automation/test-rhel.sh#L55

Sure, I'm fine with the value. It's just that it deserved a mention, exactly like this comment, but in the commit message!

**Comment by MarSik** [comment]
It is also the value we had before in the shell script.

**Comment by MarSik** [comment]
@ksimon1 Also, please make the commit message lines shorter.

File: automation/connect_to_rhel_console.exp [hunk 1]
```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/expect -f
 
+set timeout 300
+timeout {exit 1 }
```
Comment (MarSik) [inline automation/connect_to_rhel_console.exp]: I do not think it belongs here

File: automation/connect_to_rhel_console.exp [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
+
 set kubeconfig [lindex $argv 0]
 set vm_name [lindex $argv 1]
+
 spawn ./virtctl --kubeconfig=$kubeconfig console $vm_name
-expect "Successfully connected to*\r"
-send -- "\r"
-expect "*login*\r"
+expect "*Welcome*\r"
```
Comment (MarSik) [inline automation/connect_to_rhel_console.exp]: Reading the man page suggests:
expect timeout { exit 1 } "*Welcome*\r"

Comment (MarSik) [reply automation/connect_to_rhel_console.exp]: Or using the default that should catch both timeout and EOF

`expect default { exit 1 } "Welcome\r"`

**Pull request merged**